LOGGING/test/EtwCollector.Tests/EtwTelemetryModuleTests.cs

```diff
@@ -487,7 +492,11 @@ public async Task ReportsHierarchicalActivities()
                     && string.Equals(eventName, "Request/Stop", StringComparison.Ordinal));
                 Assert.IsNotNull(requestStopEvent, "Request/Stop event not found");
                 Assert.IsTrue(requestStopEvent.Properties.TryGetValue("ActivityID", out string activityID), "Event does not have ActivityID property");
+#if NETCOREAPP
```

Comment:
@TimothyMothra , don't know if that's expected. But, for netcoreapp, the guid has a very different value, generating an activityID != 0. So, changing this, the test started to pass.

Reply:
Please dont add netcore tests for this proj.

Reply:
ok!